stdlib/private/SwiftPrivateThreadExtras/ThreadBarriers.swift

```diff
@@ -12,7 +12,7 @@
 
 #if os(macOS) || os(iOS) || os(watchOS) || os(tvOS)
 import Darwin
-#elseif os(Linux) || os(FreeBSD) || os(PS4) || os(Android) || os(Cygwin) || os(Haiku)
+#elseif os(Linux) || os(FreeBSD) || os(PS4) || os(Android) || os(Cygwin) || os(Haiku) || os(OpenBSD)
```

Comment:
```suggestion
#elseif os(Linux) || os(FreeBSD) || os(OpenBSD) || os(PS4) || os(Android) || os(Cygwin) || os(Haiku)
```

...maybe, just for consistency with the rest of these? (Inconsequential nit.)

Reply:
Good catch.